diff --git a/SimPE.Wizardbase/SimPE.Wizardbase.csproj b/SimPE.Wizardbase/SimPE.Wizardbase.csproj
index cc01c4c..5df03d3 100644
--- a/SimPE.Wizardbase/SimPE.Wizardbase.csproj
+++ b/SimPE.Wizardbase/SimPE.Wizardbase.csproj
@@ -6,10 +6,7 @@
     <UseWindowsForms>true</UseWindowsForms>
     <ImportWindowsDesktopTargets>true</ImportWindowsDesktopTargets>
     <StartupObject />
-    <PostBuildEvent>if not exist "$(SolutionDir)bin\$(Configuration)\" mkdir "$(SolutionDir)bin\$(Configuration)\"
-if not exist "$(SolutionDir)SimPE.Main\bin\$(Configuration)\" mkdir "$(SolutionDir)SimPE.Main\bin\$(Configuration)\"
-copy "$(OutDir)$(AssemblyName).dll" "$(SolutionDir)bin\$(Configuration)\" /Y
-copy "$(OutDir)$(AssemblyName).dll" "$(SolutionDir)SimPE.Main\bin\$(Configuration)\" /Y</PostBuildEvent>
+    <PostBuildEvent />
     <Configurations>Debug;Release;x86</Configurations>
     <AssemblyName>simpe.wizardbase</AssemblyName>
   </PropertyGroup>
