diff --git a/ColorBinningTool/MainForm.designer.cs b/ColorBinningTool/MainForm.designer.cs
index 7fe123d..531d0f5 100644
--- a/ColorBinningTool/MainForm.designer.cs
+++ b/ColorBinningTool/MainForm.designer.cs
@@ -46,7 +46,7 @@ namespace SimPe.Plugin.UI
             SimPe.Plugin.HairtoneSettings hairtoneSettings1 = new SimPe.Plugin.HairtoneSettings();
             this.tcMain = new System.Windows.Forms.TabControl();
             this.imageList1 = new System.Windows.Forms.ImageList(this.components);
-            this.dlgOpenPackageFile = new System.Windows.Forms.OpenFileDialog();
+            this.dlgOpenPackageFile = new System.Windows.Forms.OpenFileDialog() { AutoUpgradeEnabled = false };
             this.pbTexturePreview = new System.Windows.Forms.PictureBox();
             this.toolTip1 = new System.Windows.Forms.ToolTip(this.components);
             this.cbDeleter = new System.Windows.Forms.CheckBox();
@@ -54,7 +54,7 @@ namespace SimPe.Plugin.UI
             this.llGuid = new System.Windows.Forms.LinkLabel();
             this.box = new SimPe.Plugin.GeneticCategorizer(this.components);
             this.meshTable = new SimPe.Plugin.MeshTable(this.components);
-            this.dlgSavePackageFile = new System.Windows.Forms.SaveFileDialog();
+            this.dlgSavePackageFile = new System.Windows.Forms.SaveFileDialog() { AutoUpgradeEnabled = false };
             this.cmListActions = new System.Windows.Forms.ContextMenuStrip();
             this.miOpenPackage = new System.Windows.Forms.ToolStripMenuItem();
             this.miMoveTo = new System.Windows.Forms.ToolStripMenuItem();
diff --git a/SimPE.3IDR/NmapForm.cs b/SimPE.3IDR/NmapForm.cs
index f560c79..33485e6 100644
--- a/SimPE.3IDR/NmapForm.cs
+++ b/SimPE.3IDR/NmapForm.cs
@@ -102,7 +102,7 @@ namespace SimPe.Plugin
             this.tbgroup = new System.Windows.Forms.TextBox();
             this.llcommit = new System.Windows.Forms.LinkLabel();
             this.lblist = new System.Windows.Forms.ListBox();
-            this.sfd = new System.Windows.Forms.SaveFileDialog();
+            this.sfd = new System.Windows.Forms.SaveFileDialog() { AutoUpgradeEnabled = false };
             this.btnCommit = new System.Windows.Forms.Button();
             this.panel3.SuspendLayout();
             this.wrapperPanel.SuspendLayout();
diff --git a/SimPE.AltSurgery/Surgery.cs b/SimPE.AltSurgery/Surgery.cs
index b1f1924..7e59ec8 100644
--- a/SimPE.AltSurgery/Surgery.cs
+++ b/SimPE.AltSurgery/Surgery.cs
@@ -114,14 +114,14 @@ namespace SimPe.Plugin
 			this.llusearche = new System.Windows.Forms.LinkLabel();
 			this.pbarche = new System.Windows.Forms.PictureBox();
 			this.label3 = new System.Windows.Forms.Label();
-			this.sfd = new System.Windows.Forms.SaveFileDialog();
+			this.sfd = new System.Windows.Forms.SaveFileDialog() { AutoUpgradeEnabled = false };
 			this.toolTip1 = new System.Windows.Forms.ToolTip(this.components);
 			this.cbskin = new System.Windows.Forms.CheckBox();
 			this.groupBox3 = new System.Windows.Forms.GroupBox();
 			this.lvskin = new System.Windows.Forms.ListView();
 			this.iskin = new System.Windows.Forms.ImageList(this.components);
 			this.ctlLoadPackage = new System.Windows.Forms.LinkLabel();
-			this.opd = new System.Windows.Forms.OpenFileDialog();
+			this.opd = new System.Windows.Forms.OpenFileDialog() { AutoUpgradeEnabled = false };
 			this.groupBox1.SuspendLayout();
 			this.groupBox2.SuspendLayout();
 			this.groupBox3.SuspendLayout();
diff --git a/SimPE.Downloads/LoadSims2PackTool.cs b/SimPE.Downloads/LoadSims2PackTool.cs
index 523659a..5f14fc3 100644
--- a/SimPE.Downloads/LoadSims2PackTool.cs
+++ b/SimPE.Downloads/LoadSims2PackTool.cs
@@ -48,7 +48,7 @@ namespace SimPe.Plugin.Tool
         {
             if (!ChangeEnabledStateEventHandler(sender, es)) return;
 
-            System.Windows.Forms.OpenFileDialog ofd = new System.Windows.Forms.OpenFileDialog();
+            System.Windows.Forms.OpenFileDialog ofd = new System.Windows.Forms.OpenFileDialog() { AutoUpgradeEnabled = false };
             ofd.Filter = SimPe.ExtensionProvider.BuildFilterString(new SimPe.ExtensionType[] { ExtensionType.Sim2Pack, ExtensionType.AllFiles });
             if (ofd.ShowDialog() == System.Windows.Forms.DialogResult.OK)
             {
diff --git a/SimPE.Downloads/SaveSims2Pack.cs b/SimPE.Downloads/SaveSims2Pack.cs
index 2d2fa38..b83e4b3 100644
--- a/SimPE.Downloads/SaveSims2Pack.cs
+++ b/SimPE.Downloads/SaveSims2Pack.cs
@@ -102,11 +102,11 @@ namespace SimPe.Packages
             this.label5 = new System.Windows.Forms.Label();
             this.label9 = new System.Windows.Forms.Label();
             this.btadd = new System.Windows.Forms.Button();
-            this.ofd = new System.Windows.Forms.OpenFileDialog();
+            this.ofd = new System.Windows.Forms.OpenFileDialog() { AutoUpgradeEnabled = false };
             this.label6 = new System.Windows.Forms.Label();
             this.tbflname = new System.Windows.Forms.TextBox();
             this.btbrowse = new System.Windows.Forms.Button();
-            this.sfd = new System.Windows.Forms.SaveFileDialog();
+            this.sfd = new System.Windows.Forms.SaveFileDialog() { AutoUpgradeEnabled = false };
             this.btdelete = new System.Windows.Forms.Button();
             this.btsave = new System.Windows.Forms.Button();
             this.button4 = new System.Windows.Forms.Button();
diff --git a/SimPE.Filehandlers/Elements.cs b/SimPE.Filehandlers/Elements.cs
index 3e33e77..b3321e8 100644
--- a/SimPE.Filehandlers/Elements.cs
+++ b/SimPE.Filehandlers/Elements.cs
@@ -1595,7 +1595,7 @@ namespace SimPe.PackedFiles.UserInterface
 		private void btPicExport_Click(object sender, System.EventArgs e)
 		{
 			SimPe.PackedFiles.Wrapper.Picture wrp = (SimPe.PackedFiles.Wrapper.Picture)picwrapper;
-			System.Windows.Forms.SaveFileDialog sfd = new SaveFileDialog();
+			System.Windows.Forms.SaveFileDialog sfd = new SaveFileDialog() { AutoUpgradeEnabled = false };
 			sfd.Filter = "Image (*.png) | *.png";
 
 			if (sfd.ShowDialog()==DialogResult.OK)
diff --git a/SimPE.Filehandlers/OLuaUI.cs b/SimPE.Filehandlers/OLuaUI.cs
index 6e30cca..e882c7d 100644
--- a/SimPE.Filehandlers/OLuaUI.cs
+++ b/SimPE.Filehandlers/OLuaUI.cs
@@ -293,7 +293,7 @@ namespace SimPe.PackedFiles.UserInterface
 
 		private void btSave_Click(object sender, System.EventArgs e)
 		{
-			SaveFileDialog sfd = new SaveFileDialog();
+			SaveFileDialog sfd = new SaveFileDialog() { AutoUpgradeEnabled = false };
 			sfd.Filter = SimPe.ExtensionProvider.BuildFilterString(new SimPe.ExtensionType[] {SimPe.ExtensionType.LuaScript, SimPe.ExtensionType.AllFiles});
 			sfd.FileName = Wrapper.FileName;
 			if (sfd.ShowDialog()==DialogResult.OK)			
@@ -302,7 +302,7 @@ namespace SimPe.PackedFiles.UserInterface
 
 		private void btLoad_Click(object sender, System.EventArgs e)
 		{
-			OpenFileDialog ofd = new OpenFileDialog();
+			OpenFileDialog ofd = new OpenFileDialog() { AutoUpgradeEnabled = false };
 			ofd.Filter = SimPe.ExtensionProvider.BuildFilterString(new SimPe.ExtensionType[] {SimPe.ExtensionType.LuaScript, SimPe.ExtensionType.AllFiles});
 			ofd.FileName = Wrapper.FileName;
 			if (ofd.ShowDialog()==DialogResult.OK)			
@@ -324,7 +324,7 @@ namespace SimPe.PackedFiles.UserInterface
 
 		private void button2_Click(object sender, System.EventArgs e)
 		{
-			SaveFileDialog sfd = new SaveFileDialog();
+			SaveFileDialog sfd = new SaveFileDialog() { AutoUpgradeEnabled = false };
 			sfd.Filter = SimPe.ExtensionProvider.BuildFilterString(new SimPe.ExtensionType[] {SimPe.ExtensionType.LuaScript, SimPe.ExtensionType.AllFiles});
 			sfd.FileName = Wrapper.FileName;
 			if (sfd.ShowDialog()==DialogResult.OK)	
diff --git a/SimPE.GMDCExporterbase/AnimMeshBlockControl.cs b/SimPE.GMDCExporterbase/AnimMeshBlockControl.cs
index 6c7a46b..b6daa71 100644
--- a/SimPE.GMDCExporterbase/AnimMeshBlockControl.cs
+++ b/SimPE.GMDCExporterbase/AnimMeshBlockControl.cs
@@ -363,7 +363,7 @@ namespace SimPe.Plugin.Anim
 				GeometryDataContainer gdc = (GeometryDataContainer)gmdc.Blocks[0];
 				gdc.LinkedAnimation = ab1;
 
-				fGeometryDataContainer.StartExport(new System.Windows.Forms.SaveFileDialog(), gdc, ".txt", gdc.Groups, (SimPe.Plugin.Gmdc.ElementSorting)fGeometryDataContainer.DefaultSelectedAxisIndex, this.cbCorrect.Checked);
+				fGeometryDataContainer.StartExport(new System.Windows.Forms.SaveFileDialog() { AutoUpgradeEnabled = false }, gdc, ".txt", gdc.Groups, (SimPe.Plugin.Gmdc.ElementSorting)fGeometryDataContainer.DefaultSelectedAxisIndex, this.cbCorrect.Checked);
 			} 
 			else 
 			{
@@ -380,7 +380,7 @@ namespace SimPe.Plugin.Anim
 				GeometryDataContainer gdc = (GeometryDataContainer)gmdc.Blocks[0];
 				gdc.LinkedAnimation = ab1;
 
-				fGeometryDataContainer.StartImport(new System.Windows.Forms.OpenFileDialog(), gdc, ".txt", (SimPe.Plugin.Gmdc.ElementSorting)fGeometryDataContainer.DefaultSelectedAxisIndex, true);
+				fGeometryDataContainer.StartImport(new System.Windows.Forms.OpenFileDialog() { AutoUpgradeEnabled = false }, gdc, ".txt", (SimPe.Plugin.Gmdc.ElementSorting)fGeometryDataContainer.DefaultSelectedAxisIndex, true);
 				ab1.Parent.Changed = true;
 				this.RefreshData();
 			} 
diff --git a/SimPE.GMDCExporterbase/fAnimResourceConst.cs b/SimPE.GMDCExporterbase/fAnimResourceConst.cs
index 87e4431..515e35b 100644
--- a/SimPE.GMDCExporterbase/fAnimResourceConst.cs
+++ b/SimPE.GMDCExporterbase/fAnimResourceConst.cs
@@ -480,7 +480,7 @@ namespace SimPe.Plugin.Anim
 		{
 
 			AnimationMeshBlock ab1 = (AnimationMeshBlock)tv.SelectedNode.Tag;
-			SaveFileDialog ofd = new SaveFileDialog();
+			SaveFileDialog ofd = new SaveFileDialog() { AutoUpgradeEnabled = false };
 			ofd.Filter = "TextFile (*.txt)|*.txt|All Files (*.*)|*.*";
 			if (ofd.ShowDialog() == DialogResult.OK)
 			{
@@ -516,7 +516,7 @@ namespace SimPe.Plugin.Anim
         {
             nah = true;
             AnimationMeshBlock ab1 = (AnimationMeshBlock)tv.SelectedNode.Tag;
-            OpenFileDialog ofd = new OpenFileDialog();
+            OpenFileDialog ofd = new OpenFileDialog() { AutoUpgradeEnabled = false };
             ofd.Filter = "Text files (*.txt)|*.txt";
             if (ofd.ShowDialog() == DialogResult.OK)
             {
diff --git a/SimPE.GMDCExporterbase/fGeometryDataContainer.cs b/SimPE.GMDCExporterbase/fGeometryDataContainer.cs
index d89879b..c510104 100644
--- a/SimPE.GMDCExporterbase/fGeometryDataContainer.cs
+++ b/SimPE.GMDCExporterbase/fGeometryDataContainer.cs
@@ -369,9 +369,9 @@ namespace SimPe.Plugin
             this.linkLabel5 = new System.Windows.Forms.LinkLabel();
             this.linkLabel4 = new System.Windows.Forms.LinkLabel();
             this.lb_subsets = new System.Windows.Forms.ListBox();
-            this.sfd = new System.Windows.Forms.SaveFileDialog();
+            this.sfd = new System.Windows.Forms.SaveFileDialog() { AutoUpgradeEnabled = false };
             this.cd = new System.Windows.Forms.ColorDialog();
-            this.ofd = new System.Windows.Forms.OpenFileDialog();
+            this.ofd = new System.Windows.Forms.OpenFileDialog() { AutoUpgradeEnabled = false };
             this.tabControl1.SuspendLayout();
             this.tGeometryDataContainer.SuspendLayout();
             this.groupBox1.SuspendLayout();
diff --git a/SimPE.GraphControl/AnimatedDesigner.cs b/SimPE.GraphControl/AnimatedDesigner.cs
index 6b27c19..03e3687 100644
--- a/SimPE.GraphControl/AnimatedDesigner.cs
+++ b/SimPE.GraphControl/AnimatedDesigner.cs
@@ -139,7 +139,7 @@ namespace Ambertation.Windows.Forms
 			IComponentChangeService c = (IComponentChangeService)
 				GetService(typeof(IComponentChangeService));
 
-			System.Windows.Forms.OpenFileDialog ofd = new OpenFileDialog();
+			System.Windows.Forms.OpenFileDialog ofd = new OpenFileDialog() { AutoUpgradeEnabled = false };
 			ofd.Filter = "Images (*.bmp;*.png;*.jpg;*.gif;*.wmf)|*.bmp;*.png;*.jpg;*.gif;*.wmf";
 			ofd.Multiselect = true;
 			if (ofd.ShowDialog()==DialogResult.OK) 
diff --git a/SimPE.Main/ActionExtract.cs b/SimPE.Main/ActionExtract.cs
index 571960d..56f2363 100644
--- a/SimPE.Main/ActionExtract.cs
+++ b/SimPE.Main/ActionExtract.cs
@@ -45,7 +45,7 @@ namespace SimPe.Actions.Default
 			name = name.Replace(" ", "").Replace(":", "_").Replace(@"\", "_");
 			if (!multi) 
 			{
-				System.Windows.Forms.SaveFileDialog sfd = new System.Windows.Forms.SaveFileDialog();
+				System.Windows.Forms.SaveFileDialog sfd = new System.Windows.Forms.SaveFileDialog() { AutoUpgradeEnabled = false };
 				sfd.FileName = name;
 				sfd.Filter =  ExtensionProvider.BuildFilterString(
 					new SimPe.ExtensionType[] {
diff --git a/SimPE.Main/ActionReplace.cs b/SimPE.Main/ActionReplace.cs
index 54b425b..3a9cef5 100644
--- a/SimPE.Main/ActionReplace.cs
+++ b/SimPE.Main/ActionReplace.cs
@@ -43,7 +43,7 @@ namespace SimPe.Actions.Default
 		/// <returns></returns>
 		protected SimPe.Collections.PackedFileDescriptors LoadDescriptors(bool add) 
 		{
-			System.Windows.Forms.OpenFileDialog ofd = new System.Windows.Forms.OpenFileDialog();
+			System.Windows.Forms.OpenFileDialog ofd = new System.Windows.Forms.OpenFileDialog() { AutoUpgradeEnabled = false };
 			if (!add) 
 			{
 				ofd.Filter =  ExtensionProvider.BuildFilterString(
diff --git a/SimPE.Main/AddExtTool.cs b/SimPE.Main/AddExtTool.cs
index 8040309..bf95907 100644
--- a/SimPE.Main/AddExtTool.cs
+++ b/SimPE.Main/AddExtTool.cs
@@ -99,7 +99,7 @@ namespace SimPe
 			this.tbfile = new System.Windows.Forms.TextBox();
 			this.tbattr = new System.Windows.Forms.TextBox();
 			this.button2 = new System.Windows.Forms.Button();
-			this.ofd = new System.Windows.Forms.OpenFileDialog();
+			this.ofd = new System.Windows.Forms.OpenFileDialog() { AutoUpgradeEnabled = false };
 			this.tbtype = new System.Windows.Forms.TextBox();
 			this.cbtypes = new System.Windows.Forms.ComboBox();
 			this.SuspendLayout();
diff --git a/SimPE.Main/CustomFileTableItemDialog.cs b/SimPE.Main/CustomFileTableItemDialog.cs
index 6a1bec7..906393c 100644
--- a/SimPE.Main/CustomFileTableItemDialog.cs
+++ b/SimPE.Main/CustomFileTableItemDialog.cs
@@ -126,7 +126,7 @@ namespace SimPe
 
         private void BrowseFile_Click(object sender, EventArgs e)
         {
-            using (var ofd = new OpenFileDialog())
+            using (var ofd = new OpenFileDialog() { AutoUpgradeEnabled = false })
             {
                 ofd.Filter = "Sims 2 packages (*.package)|*.package|All files (*.*)|*.*";
                 ofd.CheckFileExists = true;
diff --git a/SimPE.Main/FileTableItemForm.cs b/SimPE.Main/FileTableItemForm.cs
index 6d3e401..a3161a4 100644
--- a/SimPE.Main/FileTableItemForm.cs
+++ b/SimPE.Main/FileTableItemForm.cs
@@ -114,7 +114,7 @@ namespace SimPe
             this.label2 = new System.Windows.Forms.Label();
             this.label1 = new System.Windows.Forms.Label();
             this.fbd = new System.Windows.Forms.FolderBrowserDialog();
-            this.ofd = new System.Windows.Forms.OpenFileDialog();
+            this.ofd = new System.Windows.Forms.OpenFileDialog() { AutoUpgradeEnabled = false };
             this.xpGradientPanel1.SuspendLayout();
             this.SuspendLayout();
             // 
diff --git a/SimPE.Main/Main.Designer.cs b/SimPE.Main/Main.Designer.cs
index 2eb1836..6e8ccec 100644
--- a/SimPE.Main/Main.Designer.cs
+++ b/SimPE.Main/Main.Designer.cs
@@ -176,7 +176,7 @@ namespace SimPe
             this.tbWindow = new System.Windows.Forms.ToolStrip();
             this.tbAction = new System.Windows.Forms.ToolStrip();
             this.dockCenter = new Ambertation.Windows.Forms.DockContainer();
-            this.ofd = new System.Windows.Forms.OpenFileDialog();
+            this.ofd = new System.Windows.Forms.OpenFileDialog() { AutoUpgradeEnabled = false };
             this.miNew = new System.Windows.Forms.ToolStripMenuItem();
             this.miOpen = new System.Windows.Forms.ToolStripMenuItem();
             this.miSave = new System.Windows.Forms.ToolStripMenuItem();
@@ -216,7 +216,7 @@ namespace SimPe
             this.iAnim = new System.Windows.Forms.ImageList(this.components);
             this.xpGradientPanel3 = new System.Windows.Forms.Panel();
             this.xpGradientPanel5 = new System.Windows.Forms.Panel();
-            this.sfd = new System.Windows.Forms.SaveFileDialog();
+            this.sfd = new System.Windows.Forms.SaveFileDialog() { AutoUpgradeEnabled = false };
             this.waitControl1 = new SimPe.WaitControl();
             this.resourceViewManager1 = new SimPe.Windows.Forms.ResourceViewManager();
             this.tsmiStopWaiting = new System.Windows.Forms.ToolStripMenuItem();
diff --git a/SimPE.Main/OptionForm.Designer.cs b/SimPE.Main/OptionForm.Designer.cs
index 56f7bf8..287dcca 100644
--- a/SimPE.Main/OptionForm.Designer.cs
+++ b/SimPE.Main/OptionForm.Designer.cs
@@ -166,7 +166,7 @@ namespace SimPe
             this.button4 = new System.Windows.Forms.Button();
             this.tbdds = new System.Windows.Forms.TextBox();
             this.label5 = new System.Windows.Forms.Label();
-            this.ofd = new System.Windows.Forms.OpenFileDialog();
+            this.ofd = new System.Windows.Forms.OpenFileDialog() { AutoUpgradeEnabled = false };
             this.tbep1 = new System.Windows.Forms.TextBox();
             this.button5 = new System.Windows.Forms.Button();
             this.toolTip1 = new System.Windows.Forms.ToolTip(this.components);
diff --git a/SimPE.PluginDockBox/Report.cs b/SimPE.PluginDockBox/Report.cs
index 81c5c88..71ff8c2 100644
--- a/SimPE.PluginDockBox/Report.cs
+++ b/SimPE.PluginDockBox/Report.cs
@@ -91,7 +91,7 @@ namespace SimPe.Plugin.Tool
             this.xpGradientPanel1 = new System.Windows.Forms.Panel();
             this.button1 = new System.Windows.Forms.Button();
             this.rtb = new System.Windows.Forms.RichTextBox();
-            this.sfd = new System.Windows.Forms.SaveFileDialog();
+            this.sfd = new System.Windows.Forms.SaveFileDialog() { AutoUpgradeEnabled = false };
             this.xpGradientPanel1.SuspendLayout();
             this.SuspendLayout();
             // 
diff --git a/SimPE.RCOL/DDSTool.cs b/SimPE.RCOL/DDSTool.cs
index 890b917..321331f 100644
--- a/SimPE.RCOL/DDSTool.cs
+++ b/SimPE.RCOL/DDSTool.cs
@@ -98,7 +98,7 @@ namespace SimPe.Plugin
             System.ComponentModel.ComponentResourceManager resources = new System.ComponentModel.ComponentResourceManager(typeof(DDSTool));
             this.pb = new System.Windows.Forms.PictureBox();
             this.linkLabel1 = new System.Windows.Forms.LinkLabel();
-            this.ofd = new System.Windows.Forms.OpenFileDialog();
+            this.ofd = new System.Windows.Forms.OpenFileDialog() { AutoUpgradeEnabled = false };
             this.groupBox1 = new System.Windows.Forms.GroupBox();
             this.cbformat = new System.Windows.Forms.ComboBox();
             this.label6 = new System.Windows.Forms.Label();
diff --git a/SimPE.RCOL/tMaterialDefinitionProperties.cs b/SimPE.RCOL/tMaterialDefinitionProperties.cs
index bb752e3..3eae760 100644
--- a/SimPE.RCOL/tMaterialDefinitionProperties.cs
+++ b/SimPE.RCOL/tMaterialDefinitionProperties.cs
@@ -118,8 +118,8 @@ namespace SimPe.Plugin.TabPage
             this.btnImport = new System.Windows.Forms.Button();
             this.btnExport = new System.Windows.Forms.Button();
             this.btnMerge = new System.Windows.Forms.Button();
-            this.saveFileDialog1 = new System.Windows.Forms.SaveFileDialog();
-            this.openFileDialog1 = new System.Windows.Forms.OpenFileDialog();
+            this.saveFileDialog1 = new System.Windows.Forms.SaveFileDialog() { AutoUpgradeEnabled = false };
+            this.openFileDialog1 = new System.Windows.Forms.OpenFileDialog() { AutoUpgradeEnabled = false };
             this.gbprop.SuspendLayout();
             this.SuspendLayout();
             // 
diff --git a/SimPE.Scenegraph/LifoForm.cs b/SimPE.Scenegraph/LifoForm.cs
index 0a70532..ffd5ba3 100644
--- a/SimPE.Scenegraph/LifoForm.cs
+++ b/SimPE.Scenegraph/LifoForm.cs
@@ -105,8 +105,8 @@ namespace SimPe.Plugin
             this.panel2 = new System.Windows.Forms.Panel();
             this.btex = new System.Windows.Forms.Button();
             this.btim = new System.Windows.Forms.Button();
-            this.sfd = new System.Windows.Forms.SaveFileDialog();
-            this.ofd = new System.Windows.Forms.OpenFileDialog();
+            this.sfd = new System.Windows.Forms.SaveFileDialog() { AutoUpgradeEnabled = false };
+            this.ofd = new System.Windows.Forms.OpenFileDialog() { AutoUpgradeEnabled = false };
             this.LifoPanel.SuspendLayout();
             this.panel1.SuspendLayout();
             ((System.ComponentModel.ISupportInitialize)(this.pb)).BeginInit();
diff --git a/SimPE.Scenegraph/TxtrForm.cs b/SimPE.Scenegraph/TxtrForm.cs
index 378b911..c24146b 100644
--- a/SimPE.Scenegraph/TxtrForm.cs
+++ b/SimPE.Scenegraph/TxtrForm.cs
@@ -122,8 +122,8 @@ namespace SimPe.Plugin
             this.btex = new System.Windows.Forms.Button();
             this.btim = new System.Windows.Forms.Button();
             this.label1 = new System.Windows.Forms.Label();
-            this.sfd = new System.Windows.Forms.SaveFileDialog();
-            this.ofd = new System.Windows.Forms.OpenFileDialog();
+            this.sfd = new System.Windows.Forms.SaveFileDialog() { AutoUpgradeEnabled = false };
+            this.ofd = new System.Windows.Forms.OpenFileDialog() { AutoUpgradeEnabled = false };
             this.txtrPanel.SuspendLayout();
             this.panel1.SuspendLayout();
             ((System.ComponentModel.ISupportInitialize)(this.pb)).BeginInit();
diff --git a/SimPE.ToolBoxWorkshops/OWDockForm.cs b/SimPE.ToolBoxWorkshops/OWDockForm.cs
index d45ba44..b9a3b26 100644
--- a/SimPE.ToolBoxWorkshops/OWDockForm.cs
+++ b/SimPE.ToolBoxWorkshops/OWDockForm.cs
@@ -171,7 +171,7 @@ namespace SimPe.Plugin.Tool.Dockable
             this.biAbort = new System.Windows.Forms.ToolStripButton();
             this.biCatalog = new System.Windows.Forms.ToolStripButton();
 			this.ilist = new System.Windows.Forms.ImageList(this.components);
-			this.sfd = new System.Windows.Forms.SaveFileDialog();
+			this.sfd = new System.Windows.Forms.SaveFileDialog() { AutoUpgradeEnabled = false };
 			this.rightSandDock.SuspendLayout();
 			this.dcObjectWorkshop.SuspendLayout();
 			this.xpGradientPanel1.SuspendLayout();
@@ -919,7 +919,7 @@ namespace SimPe.Plugin.Tool.Dockable
 
 		private void button2_Click(object sender, System.EventArgs e)
 		{
-			OpenFileDialog ofd = new OpenFileDialog();
+			OpenFileDialog ofd = new OpenFileDialog() { AutoUpgradeEnabled = false };
 			ofd.Filter = ExtensionProvider.BuildFilterString(
 				new SimPe.ExtensionType[] {
 											  SimPe.ExtensionType.Package,
diff --git a/SimPE.ToolBoxWorkshops/ObjectWorkshopHelper.cs b/SimPE.ToolBoxWorkshops/ObjectWorkshopHelper.cs
index 3fbd87f..0e6fc68 100644
--- a/SimPE.ToolBoxWorkshops/ObjectWorkshopHelper.cs
+++ b/SimPE.ToolBoxWorkshops/ObjectWorkshopHelper.cs
@@ -452,7 +452,7 @@ namespace SimPe.Plugin.Tool.Dockable
 				if (Message.Show(Localization.Manager.GetString("OW_Warning"), "Warning", MessageBoxButtons.YesNo)==DialogResult.No) return package;
 			}
 
-			SaveFileDialog sfd = new SaveFileDialog();
+			SaveFileDialog sfd = new SaveFileDialog() { AutoUpgradeEnabled = false };
 			sfd.Filter = ExtensionProvider.BuildFilterString(
 				new SimPe.ExtensionType[] {
 											  SimPe.ExtensionType.Package,
@@ -487,7 +487,7 @@ namespace SimPe.Plugin.Tool.Dockable
 				if (Message.Show(Localization.Manager.GetString("OW_Warning"), "Warning", MessageBoxButtons.YesNo)==DialogResult.No) return package;
 			}
 
-			SaveFileDialog sfd = new SaveFileDialog();
+			SaveFileDialog sfd = new SaveFileDialog() { AutoUpgradeEnabled = false };
 			sfd.Filter = ExtensionProvider.BuildFilterString(
 				new SimPe.ExtensionType[] {
 											  SimPe.ExtensionType.Package,
@@ -552,7 +552,7 @@ namespace SimPe.Plugin.Tool.Dockable
                     map = fo.GetNameMap(true);
                     if (map == null) return package;
 
-                    SaveFileDialog sfd = new SaveFileDialog();
+                    SaveFileDialog sfd = new SaveFileDialog() { AutoUpgradeEnabled = false };
                     sfd.Filter = ExtensionProvider.BuildFilterString(
                         new SimPe.ExtensionType[] {
 													  SimPe.ExtensionType.Package,
diff --git a/SimPE.ToolBoxWorkshops/ObjectsDock.cs b/SimPE.ToolBoxWorkshops/ObjectsDock.cs
index 0949894..81b99be 100644
--- a/SimPE.ToolBoxWorkshops/ObjectsDock.cs
+++ b/SimPE.ToolBoxWorkshops/ObjectsDock.cs
@@ -1011,7 +1011,7 @@ namespace SimPe.Plugin.Tool.Dockable
 		private void button2_Click(object sender, System.EventArgs e)
 		{
 			onlybase = false;
-			OpenFileDialog ofd = new OpenFileDialog();
+			OpenFileDialog ofd = new OpenFileDialog() { AutoUpgradeEnabled = false };
 			ofd.Filter = ExtensionProvider.BuildFilterString(
 				new SimPe.ExtensionType[] {
 											  SimPe.ExtensionType.Package,
diff --git a/SimPE.ToolBoxWorkshops/PackageRepairForm.cs b/SimPE.ToolBoxWorkshops/PackageRepairForm.cs
index 938c91d..31a29d4 100644
--- a/SimPE.ToolBoxWorkshops/PackageRepairForm.cs
+++ b/SimPE.ToolBoxWorkshops/PackageRepairForm.cs
@@ -265,7 +265,7 @@ namespace SimPe.Plugin.Tool.Window
 
 		private void btBrowse_Click(object sender, System.EventArgs e)
 		{
-			OpenFileDialog ofd = new OpenFileDialog();
+			OpenFileDialog ofd = new OpenFileDialog() { AutoUpgradeEnabled = false };
 			ofd.Filter = SimPe.ExtensionProvider.BuildFilterString(new SimPe.ExtensionType[] { SimPe.ExtensionType.Package, SimPe.ExtensionType.AllFiles});
 			if (ofd.ShowDialog()==DialogResult.OK)
 				Setup(ofd.FileName);
diff --git a/SimPE.ToolBoxWorkshops/Workshop.cs b/SimPE.ToolBoxWorkshops/Workshop.cs
index 5789423..a8f7530 100644
--- a/SimPE.ToolBoxWorkshops/Workshop.cs
+++ b/SimPE.ToolBoxWorkshops/Workshop.cs
@@ -388,7 +388,7 @@ namespace SimPe.Plugin
             this.tColor = new System.Windows.Forms.TabPage();
             this.cbColorExt = new System.Windows.Forms.CheckBox();
             this.cbColor = new System.Windows.Forms.CheckBox();
-            this.sfd = new System.Windows.Forms.SaveFileDialog();
+            this.sfd = new System.Windows.Forms.SaveFileDialog() { AutoUpgradeEnabled = false };
             this.tbseek = new System.Windows.Forms.TextBox();
             this.tabControl2 = new System.Windows.Forms.TabControl();
             this.tabPage1 = new System.Windows.Forms.TabPage();
@@ -398,7 +398,7 @@ namespace SimPe.Plugin
             this.tabPage2 = new System.Windows.Forms.TabPage();
             this.button1 = new System.Windows.Forms.Button();
             this.tbflname = new System.Windows.Forms.TextBox();
-            this.ofd = new System.Windows.Forms.OpenFileDialog();
+            this.ofd = new System.Windows.Forms.OpenFileDialog() { AutoUpgradeEnabled = false };
             this.groupBox1.SuspendLayout();
             ((System.ComponentModel.ISupportInitialize)(this.pb)).BeginInit();
             this.tabControl1.SuspendLayout();
diff --git a/SimPE.Toolbox/PhotoStudio.cs b/SimPE.Toolbox/PhotoStudio.cs
index 7527b6e..f692bc7 100644
--- a/SimPE.Toolbox/PhotoStudio.cs
+++ b/SimPE.Toolbox/PhotoStudio.cs
@@ -158,8 +158,8 @@ namespace SimPe.Plugin
             this.llcreate = new System.Windows.Forms.LinkLabel();
             this.label2 = new System.Windows.Forms.Label();
             this.cbprev = new System.Windows.Forms.CheckBox();
-            this.ofd = new System.Windows.Forms.OpenFileDialog();
-            this.sfd = new System.Windows.Forms.SaveFileDialog();
+            this.ofd = new System.Windows.Forms.OpenFileDialog() { AutoUpgradeEnabled = false };
+            this.sfd = new System.Windows.Forms.SaveFileDialog() { AutoUpgradeEnabled = false };
             this.panel1 = new System.Windows.Forms.Panel();
             ((System.ComponentModel.ISupportInitialize)(this.pbpreview)).BeginInit();
             this.tabControl1.SuspendLayout();
diff --git a/SimPE.Toolbox/SkinWorkshop.cs b/SimPE.Toolbox/SkinWorkshop.cs
index 40c6b21..e13ad3c 100644
--- a/SimPE.Toolbox/SkinWorkshop.cs
+++ b/SimPE.Toolbox/SkinWorkshop.cs
@@ -241,9 +241,9 @@ namespace SimPe.Plugin
             this.pb = new System.Windows.Forms.PictureBox();
             this.cbfix = new System.Windows.Forms.CheckBox();
             this.cbgid = new System.Windows.Forms.CheckBox();
-            this.sfd = new System.Windows.Forms.SaveFileDialog();
+            this.sfd = new System.Windows.Forms.SaveFileDialog() { AutoUpgradeEnabled = false };
             this.tbseek = new System.Windows.Forms.TextBox();
-            this.ofd = new System.Windows.Forms.OpenFileDialog();
+            this.ofd = new System.Windows.Forms.OpenFileDialog() { AutoUpgradeEnabled = false };
             this.panel1 = new System.Windows.Forms.Panel();
             this.taskBox2 = new System.Windows.Forms.Panel();
             this.taskBox1 = new System.Windows.Forms.Panel();
diff --git a/SimPE.Toolbox/Surgery.cs b/SimPE.Toolbox/Surgery.cs
index 02de145..d6b3ed7 100644
--- a/SimPE.Toolbox/Surgery.cs
+++ b/SimPE.Toolbox/Surgery.cs
@@ -116,7 +116,7 @@ namespace SimPe.Plugin
             this.llusearche = new System.Windows.Forms.LinkLabel();
             this.pbarche = new System.Windows.Forms.PictureBox();
             this.label3 = new System.Windows.Forms.Label();
-            this.sfd = new System.Windows.Forms.SaveFileDialog();
+            this.sfd = new System.Windows.Forms.SaveFileDialog() { AutoUpgradeEnabled = false };
             this.toolTip1 = new System.Windows.Forms.ToolTip(this.components);
             this.cbskin = new System.Windows.Forms.CheckBox();
             this.groupBox3 = new System.Windows.Forms.GroupBox();
diff --git a/SimPE.ToolboxScanner/ScannerForm.cs b/SimPE.ToolboxScanner/ScannerForm.cs
index 17ee530..97cc75a 100644
--- a/SimPE.ToolboxScanner/ScannerForm.cs
+++ b/SimPE.ToolboxScanner/ScannerForm.cs
@@ -436,7 +436,7 @@ namespace SimPe.Plugin
             this.lbname = new System.Windows.Forms.Label();
             this.llopen = new System.Windows.Forms.LinkLabel();
             this.thumb = new System.Windows.Forms.PictureBox();
-            this.sfd = new System.Windows.Forms.SaveFileDialog();
+            this.sfd = new System.Windows.Forms.SaveFileDialog() { AutoUpgradeEnabled = false };
             this.toolTip1 = new System.Windows.Forms.ToolTip(this.components);
             this.panel1 = new System.Windows.Forms.Panel();
             this.tabControl1.SuspendLayout();
diff --git a/SimPE.ToolboxScanner/ScannerPanelForm.cs b/SimPE.ToolboxScanner/ScannerPanelForm.cs
index 6097726..ba8076a 100644
--- a/SimPE.ToolboxScanner/ScannerPanelForm.cs
+++ b/SimPE.ToolboxScanner/ScannerPanelForm.cs
@@ -203,7 +203,7 @@ namespace SimPe.Plugin.Scanner
             this.label4 = new System.Windows.Forms.Label();
             this.visualStyleLinkLabel2 = new System.Windows.Forms.LinkLabel();
             this.cbref = new System.Windows.Forms.CheckBox();
-            this.sfd = new System.Windows.Forms.SaveFileDialog();
+            this.sfd = new System.Windows.Forms.SaveFileDialog() { AutoUpgradeEnabled = false };
             this.tabControl1.SuspendLayout();
             this.tabPage2.SuspendLayout();
             this.pnep.SuspendLayout();
diff --git a/Wizards of SimPe/Option.cs b/Wizards of SimPe/Option.cs
index e340e0a..ae02dd6 100644
--- a/Wizards of SimPe/Option.cs	
+++ b/Wizards of SimPe/Option.cs	
@@ -111,7 +111,7 @@ namespace SimPe.Wizards
             this.linkLabel1 = new System.Windows.Forms.LinkLabel();
             this.pbstretch = new System.Windows.Forms.PictureBox();
             this.fbd = new System.Windows.Forms.FolderBrowserDialog();
-            this.ofd = new System.Windows.Forms.OpenFileDialog();
+            this.ofd = new System.Windows.Forms.OpenFileDialog() { AutoUpgradeEnabled = false };
             ((System.ComponentModel.ISupportInitialize)(this.pbtop)).BeginInit();
             ((System.ComponentModel.ISupportInitialize)(this.pbbottom)).BeginInit();
             this.pnopt.SuspendLayout();
diff --git a/_PJSE/pjBodyMeshTool/BodyMeshExtractor.cs b/_PJSE/pjBodyMeshTool/BodyMeshExtractor.cs
index 21333e7..9f573ed 100644
--- a/_PJSE/pjBodyMeshTool/BodyMeshExtractor.cs
+++ b/_PJSE/pjBodyMeshTool/BodyMeshExtractor.cs
@@ -82,7 +82,7 @@ namespace pj
         private IPackageFile currentPackage;
         private String getFilename()
         {
-            OpenFileDialog ofd = new OpenFileDialog();
+            OpenFileDialog ofd = new OpenFileDialog() { AutoUpgradeEnabled = false };
             ofd.AddExtension = true;
             ofd.CheckFileExists = true;
             ofd.CheckPathExists = true;
diff --git a/_PJSE/pjBodyMeshTool/BodyMeshLinker.cs b/_PJSE/pjBodyMeshTool/BodyMeshLinker.cs
index 9a1b4a2..cb3528b 100644
--- a/_PJSE/pjBodyMeshTool/BodyMeshLinker.cs
+++ b/_PJSE/pjBodyMeshTool/BodyMeshLinker.cs
@@ -38,7 +38,7 @@ namespace pj
 
         private String getFilename()
         {
-            OpenFileDialog ofd = new OpenFileDialog();
+            OpenFileDialog ofd = new OpenFileDialog() { AutoUpgradeEnabled = false };
             ofd.AddExtension = true;
             ofd.CheckFileExists = true;
             ofd.CheckPathExists = true;
diff --git a/_PJSE/pjse Coder/BhavForm.cs b/_PJSE/pjse Coder/BhavForm.cs
index 8dcd1d9..00e3a40 100644
--- a/_PJSE/pjse Coder/BhavForm.cs	
+++ b/_PJSE/pjse Coder/BhavForm.cs	
@@ -2591,9 +2591,9 @@ namespace SimPe.PackedFiles.UserInterface
             bool load = sender.Equals(this.fromFileToolStripMenuItem);
             FileDialog fd;
             if (load)
-                fd = new OpenFileDialog();
+                fd = new OpenFileDialog() { AutoUpgradeEnabled = false };
             else
-                fd = new SaveFileDialog();
+                fd = new SaveFileDialog() { AutoUpgradeEnabled = false };
             fd.AddExtension = true;
             fd.CheckFileExists = load;
             fd.CheckPathExists = true;
diff --git a/_PJSE/pjse Coder/ExtractCurrent.cs b/_PJSE/pjse Coder/ExtractCurrent.cs
index b6edd3c..bb18e0b 100644
--- a/_PJSE/pjse Coder/ExtractCurrent.cs	
+++ b/_PJSE/pjse Coder/ExtractCurrent.cs	
@@ -37,7 +37,7 @@ namespace pjse
     {
         public static DialogResult Execute(SimPe.Interfaces.Plugin.AbstractWrapper wrapper, string title)
         {
-            SaveFileDialog sfd = new SaveFileDialog();
+            SaveFileDialog sfd = new SaveFileDialog() { AutoUpgradeEnabled = false };
             sfd.FileName = wrapper.FileDescriptor.ExportFileName.Replace(" ", "").Replace(":", "_").Replace(@"\", "_");
             sfd.Filter = SimPe.ExtensionProvider.BuildFilterString(
                 new SimPe.ExtensionType[] { SimPe.ExtensionType.ExtractedFile, SimPe.ExtensionType.AllFiles }
diff --git a/_PJSE/pjse Coder/StrForm.cs b/_PJSE/pjse Coder/StrForm.cs
index d4cbd4d..e36b7f9 100644
--- a/_PJSE/pjse Coder/StrForm.cs	
+++ b/_PJSE/pjse Coder/StrForm.cs	
@@ -628,7 +628,7 @@ namespace SimPe.PackedFiles.UserInterface
 
         private void StringFile(bool load)
         {
-            FileDialog fd = load ? (FileDialog)new OpenFileDialog() : (FileDialog)new SaveFileDialog();
+            FileDialog fd = load ? (FileDialog)new OpenFileDialog() { AutoUpgradeEnabled = false } : (FileDialog)new SaveFileDialog() { AutoUpgradeEnabled = false };
             fd.AddExtension = true;
             fd.CheckFileExists = load;
             fd.CheckPathExists = true;
diff --git a/__NonCore/WOSimPe - Recolor/RecolorWizardForm.cs b/__NonCore/WOSimPe - Recolor/RecolorWizardForm.cs
index b4bc8d6..b87f134 100644
--- a/__NonCore/WOSimPe - Recolor/RecolorWizardForm.cs	
+++ b/__NonCore/WOSimPe - Recolor/RecolorWizardForm.cs	
@@ -156,8 +156,8 @@ namespace SimPe.Wizards
             this.cbover = new System.Windows.Forms.CheckBox();
             this.iObjects = new System.Windows.Forms.ImageList(this.components);
             this.toolTip1 = new System.Windows.Forms.ToolTip(this.components);
-            this.sfd = new System.Windows.Forms.SaveFileDialog();
-            this.ofd = new System.Windows.Forms.OpenFileDialog();
+            this.sfd = new System.Windows.Forms.SaveFileDialog() { AutoUpgradeEnabled = false };
+            this.ofd = new System.Windows.Forms.OpenFileDialog() { AutoUpgradeEnabled = false };
             this.tabControl1.SuspendLayout();
             this.tabPage4.SuspendLayout();
             ((System.ComponentModel.ISupportInitialize)(this.pbgeneral)).BeginInit();
